keyutils/keyutils.go

```diff
@@ -0,0 +1,87 @@
+package keyutils
+
+import (
+	"crypto"
+	cryptorand "crypto/rand"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"os"
+
+	"github.com/cloudflare/cfssl/helpers"
+	"github.com/docker/swarmkit/pkcs8"
+)
+
+// FIPSEnvVar is the environment variable name for FIPS mode
+const FIPSEnvVar = "GOFIPS"
+
+// FIPSEnabled returns true if FIPS mode is enabled
+func FIPSEnabled() bool {
```

Comment:
It looks like `FIPSEnabled` is only used in this package, so it does not need to be exported.